### ISSUE TITLE
May meetup second talk

### ISSUE THREAD

**Issue by huima** [clojure-finland/home#12]

**Comment by pkalliok** [comment]
Just as a contribution to the community, I could give a talk about the history of visual programming.  It does not have especially much to do with Clojure, so if you have an excess of topics, leave this one out.

**Comment by pkalliok** [comment]
BTW, I have been very long saving cool "talk topics" or "blog topics" from different communities: http://sange.fi/~atehwa/cgi-bin/piki.cgi/hyvi%E4%20aiheita%20teoreettisen%20tietojenk%E4sittelytieteen%20alalta - some of them are quite relevant for the Clojure community and can act as an inspiration if someone wants to give a presentation for them.  (Many of those topics are ones that I could talk about.)

**Comment by mhjort** [comment]
@pkalliok I could still wait for some Clojure talk proposals. If I don't get enough proposals let's get back to this visual programming topic?

**Comment by juhokallio** [comment]
I could also present something from us ([IPRally](https://www.iprally.com), startup building patent search with knowledge graphs). We are doing everything with Clojure(Script), except the Python AI parts.

Possible subjects:
- Recursive UI with ClojureScript. Lots of Clojure logic for manipulating graphs, which might be more pain in plain JavaScript.
- Our first Clojure open source library for Google Cloud Pub/Sub, which might or might not be published at the time
- Startup software engineering with Clojure
- More general company presentation

**Comment by huima** [comment]
@pkalliok it would be really fun to have more general talks about computation too. Regarding that list of potential topics and ideas, should we open up issue or even a markdown page for people to edit bubbling under ideas / topics that could become talks eventually?

**Comment by pkalliok** [comment]
Yes, I can contribute some ideas.  As I said, my suggestion is meant when no more on-topic presentation is available.  Just remember to tell me if my talk is on anyway :)

**Comment by juhokallio** [comment]
I could have the talk in a later meetup instead, as the week will be pretty intense for me anyhow, e.g. including another presentation.

**Comment by jaakkos** [comment]
I could give a short talk about "Things Learned from Clojure Interop - There and Back Again".

**Comment by mhjort** [comment]
I have added "Things Learned from Clojure Interop - There and Back Again" to agenda.

@pkalliok Could you still do the "the history of visual programming" talk? Topic is interesting and there is  a room for that.
